tests/test_stainnorm.py

```diff
@@ -1,11 +1,65 @@
 from tiatoolbox.utils.misc import imread
 from tiatoolbox.tools.stainnorm import get_normaliser
+from tiatoolbox.tools import stainextract
 from tiatoolbox import cli
 from tiatoolbox.utils.exceptions import MethodNotSupported
 
 import pathlib
 import numpy as np
 from click.testing import CliRunner
+import pytest
+
+
+def test_stain_extract():
+    """Test stain extraction class."""
+    stain_matrix = np.array([0.65, 0.70, 0.29])
+    with pytest.raises(ValueError):
+        _ = stainextract.CustomExtractor(stain_matrix)
+
+
+def test_vectors_in_right_direction():
+    """Test if eigen vectors are corrected in the right direction."""
```

Comment:
eigenvectors all one word

Reply:
Fixed typo in this commit https://github.com/TIA-Lab/tiatoolbox/commit/999126fa1462632e8db7354d0d944fde4ec894de